app/views/event6/TypeOfProtectionView.scala.html

```diff
@@ -0,0 +1,96 @@
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_0" name="value" type="radio" value="enhancedLifetimeAllowance">
+                        <label class="govuk-label govuk-radios__label" for="value_0">
+                            @messages("typeOfProtection.enhancedLifetimeAllowance")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_1" name="value" type="radio" value="enhancedProtection">
+                        <label class="govuk-label govuk-radios__label" for="value_1">
+                            @messages("typeOfProtection.enhancedProtection")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_2" name="value" type="radio" value="fixedProtection">
+                        <label class="govuk-label govuk-radios__label" for="value_2">
+                            @messages("typeOfProtection.fixedProtection")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_3" name="value" type="radio" value="fixedProtection2014">
+                        <label class="govuk-label govuk-radios__label" for="value_3">
+                            @messages("typeOfProtection.fixedProtection2014")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_4" name="value" type="radio" value="fixedProtection2016">
+                        <label class="govuk-label govuk-radios__label" for="value_4">
+                            @messages("typeOfProtection.fixedProtection2016")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_5" name="value" type="radio" value="individualProtection2014">
+                        <label class="govuk-label govuk-radios__label" for="value_5">
+                            @messages("typeOfProtection.individualProtection2014")
+                        </label>
+                    </div>
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="value_6" name="value" type="radio" value="individualProtection2016">
+                        <label class="govuk-label govuk-radios__label" for="value_6">
+                            @messages("typeOfProtection.individualProtection2016")
+                        </label>
+                    </div>
+                </div>
+            </fieldset>
+        </div>
```

Comment:
I think we've discussed this before but there's a massive refactoring we can do here. Here's the code snippet I wrote for this:

```
                @for(option <- TypeOfProtection.options) {
                <div class="govuk-radios__item">
                    <input class="govuk-radios__input" id=@(option.id) name="value" type="radio" value=@(option.value)">
                    <label class="govuk-label govuk-radios__label" for=@(option.id)>
                       @(option.content.asHtml)
                    </label>
                </div>
                }
```

Reply:
Resolved as per discussion